src/pkg/packager/create.go

```diff
@@ -453,7 +466,7 @@ func (p *Packager) addComponent(component types.ZarfComponent) (*types.Component
 			// Pull all the references if there is no `@` in the string.
 			gitCfg := git.NewWithSpinner(p.cfg.State.GitServer, spinner)
 			if err := gitCfg.Pull(url, componentPath.Repos); err != nil {
-				return nil, fmt.Errorf("unable to pull git repo %s: %w", url, err)
+				return nil, fmt.Errorf("unable to pull git repoURL %s: %w", url, err)
```

Comment:
```suggestion
				return nil, fmt.Errorf("unable to pull git repo URL %s: %w", url, err)
```
Was this supposed to be changed to repoURL? (no space)